### PR TITLE
Update to 22.12.0.

### PR DESCRIPTION
**Pull request by jezdez**
This is dependent on the conda 22.11.0 release https://github.com/AnacondaRecipes/conda-feedstock/pull/17

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
```
conda.exceptions.ResolvePackageNotFound:
  - conda[version='>=22.11']
```

**Comment by jezdez** [comment]
Ah, closing and reopening again :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,22 +24,22 @@ requirements:
     - flit-core >=3.2,<4
   run:
     - python
-    - libmambapy >=0.23
-    - conda >=4.13
-    - importlib_metadata
+    - libmambapy >=1.0
+    - conda >=22.11
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:28]: `conda >=22.11.0` would be correct

Comment (jezdez) [inline recipe/meta.yaml:28]:
```suggestion
    - conda >=22.11.0
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -24,22 +24,22 @@ requirements:
     - flit-core >=3.2,<4
   run:
     - python
-    - libmambapy >=0.23
-    - conda >=4.13
-    - importlib_metadata
+    - libmambapy >=1.0
```
Comment (jezdez) [inline recipe/meta.yaml:27]:
```suggestion
    - libmambapy >=1.0.0
```

Comment (skupr-anaconda) [inline recipe/meta.yaml]: I guess the upstream `libmambapy >=0.23` is outdated https://github.com/conda/conda-libmamba-solver/blob/990b25213b4f6c7a39c96533c928a8d690d12a86/pyproject.toml#L27

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (jezdez) [reply recipe/meta.yaml]: https://github.com/conda/conda-libmamba-solver/pull/90

Comment (jezdez) [reply recipe/meta.yaml]: I'll do a 22.11.1 with that

**Comment by jezdez** [comment]
Update to 22.12.0!

**Review by dholth — APPROVED** [review]
LGTM

**Pull request merged**